.gitignore

```diff
@@ -36,6 +36,7 @@ bower
 node_modules
 /udata_gouvfr/theme/static/
 /udata_gouvfr/theme/manifest.json
+package-lock.json
```

Comment:
nope, sorry :-) this a safety feature, a canari in the mine: if you're adding a lock file, you're using the wrong file.

Plus we'll want to commit the lock file when it's supported.

Reply:
ok let the canari live its life :)